app/controllers/users_controller.rb

```diff
@@ -1,4 +1,5 @@
 class UsersController < ApplicationController
+  before_action :require_user
```

Comment:
This is a good piece of functionality but I wonder if there's a way to not need this in every controller?